clients/go/apis/v1alpha1/config.go

```diff
@@ -358,3 +349,24 @@ func (r *AuditRule) SetDefault() {
 		r.LogType = AuditLogRequest_DATA_ACCESS.String()
 	}
 }
+
+// Justification specifies the config used to integrate with JVS.
+type Justification struct {
+	// JVSPublicKeysEndpoint is the endpoint where public keys may be retrieved from the JVS.
+	// These will be used to validate JWT tokens that are passed in through the
+	// "justification-token" header.
+	JVSPublicKeysEndpoint string `yaml:"jvs_public_keys_endpoint,omitempty"`
```

Comment:
These seem to be missing env keys

Reply:
+1

Reply:
Do we want to change the `PublicKeyEndpont` via env variables?
(To clarify, `PublicKeyEndpont`  is field of struct [justification] https://github.com/abcxyz/lumberjack/pull/284/files#diff-be7b293b39821f8b46a4da6a72de39a29d1508d34745442b19d0028a6dbe00bcR69)

Reply:
I think my rule of thumb is to allow config setting/override with env var if possible. We only don't support env var for some repeated fields, e.g. rules.

Reply:
done